# upstream commit: https://github.com/equalsraf/neovim-qt/commit/c6ae970

From c6ae9709237b6a55a705687afe4103646f34ac60 Mon Sep 17 00:00:00 2001
From: Carlo Cabrera <30379873+carlocab@users.noreply.github.com>
Date: Sun, 5 Mar 2023 23:03:47 +0800
Subject: [PATCH] Fix finding msgpack 6+

`libmsgpackc` was renamed to `libmsgpack-c` in 6.0.0. See
msgpack/msgpack-c#1053.
---
 cmake/FindMsgpack.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/FindMsgpack.cmake b/cmake/FindMsgpack.cmake
index 0e2013f9d..41e2d7b5c 100644
--- a/cmake/FindMsgpack.cmake
+++ b/cmake/FindMsgpack.cmake
@@ -5,7 +5,7 @@ find_path(MSGPACK_INCLUDE_DIR
 )
 
 find_library(MSGPACK_LIBRARY
-    NAMES msgpack msgpackc libmsgpack.a libmsgpackc.a
+    NAMES msgpack-c msgpack msgpackc libmsgpack.a libmsgpackc.a
 )
 
 mark_as_advanced(MSGPACK_INCLUDE_DIR MSGPACK_LIBRARY)
